bindings/wasm/bindings.d.ts

```diff
@@ -144,3 +144,6 @@ declare function union(a: Manifold, b: Manifold): Manifold;
 declare function difference(a: Manifold, b: Manifold): Manifold;
 declare function intersection(a: Manifold, b: Manifold): Manifold;
 
+declare function union(manifolds: Manifold[]): Manifold;
```

Comment:
I was already thinking the a, b union, difference, intersection were feeling pretty duplicate to the add, subtract functions anyway. Shall we just remove them in favor of these vector versions?

Reply:
Yeah we can remove that.